src/t8_schemes/t8_standalone/t8_standalone_implementation.hxx

```diff
@@ -950,8 +1210,7 @@ struct t8_standalone_scheme
       /* get id in subtree of child */
       id -= sum_descendants_of_children_before;
     }
-    T8_ASSERT (id < T8_ELEMENT_NUM_CHILDREN[TEclass]);
-    element_get_child ((const t8_element_t *) el, id, (t8_element_t *) el);
```

Comment:
Please move to a separate PR with a scheme test that fails with this line included